slam-libraries/viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -314,8 +347,6 @@ void SLAMServiceImpl::CreateMap() {
         map_builder.map_builder_->pose_graph()->RunFinalOptimization();
     }
 
-    LOG(INFO) << "Finished processing offline images";
```

Comment:
Keep the log but change the text to something like "Finished optimizing final map"